app/views/user_mailer/assignment_reminder.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
```

Comment:
This title can be removed.

We should add the content type meta tag here for utf-8. I'm not sure how email clients treat encoding of the email page but better safe than sorry.